### ISSUE TITLE
Array Index Out Of Bounds Exception: 77 Crash

### ISSUE THREAD

**Issue by WenXin20** [Furgl/Stupid-Things#2]
Stupid Things v1.1
Forge v2316-1.10.2
I got this crash on game launch.
[Crash Log](https://pastebin.com/AyyRiyXA)

**Comment by Furgl** [comment]
Thank you for this report.

The line responsible for this in my code typically will not cause this type of crash. If I had to guess, I would say one of the coremods installed is messing with items in a way that checking tool materials during init causes this crash. Either way, this should be a simple fix. I attached a version of the mod that should work without crashing for you. I'll have this fix added in the next public update as well.

Let me know if this issue is still not fixed for whatever reason and I will reopen this and investigate the issue further.
[Stupid Things-1.10.2-1.1-Fix.jar.zip](https://github.com/Furgl/Stupid-Things/files/1081631/Stupid.Things-1.10.2-1.1-Fix.jar.zip)
